doc/api/cli.md

```diff
@@ -1690,6 +1697,7 @@ $ node --max-old-space-size=1536 index.js
 [REPL]: repl.md
 [ScriptCoverage]: https://chromedevtools.github.io/devtools-protocol/tot/Profiler#type-ScriptCoverage
 [Source Map]: https://sourcemaps.info/spec.html
+[Source Map v3]: https://sourcemaps.info/spec.html
```

Comment:
Seems like we could remove this and rely on the line immediately above. I think you can use `[Source Map v3][Source Map]` to use it with the link text you want.